text/0265-event-vision.md

```diff
@@ -0,0 +1,75 @@
+# Event Basics
+
+## Motivation
+
+The introduction of Events has been contentious, so we want to document and agree on a few basics.
+
+### What are OpenTelemetry Events?
+
+OpenTelemetry Events are a type of OpenTelemetry Log that requires an event name and follows a specific structure implied by that event name.
+
+They are a core concept in OpenTelemetry Semantic Conventions.
+
+### OTLP
+
+Since OpenTelemetry Events are a type of OpenTelemetry Log, they share the same OTLP log data structure and pipeline.
+
+### API
+
+OpenTelemetry should provide a (user-facing) Log API that includes the capability to emit OpenTelemetry Events.
+
+### Interoperability with Generic Logging Libraries
+
+It should be possible to send OpenTelemetry Logs from the OpenTelemetry Log API to a generic logging library (e.g., Log4j).
+This allows users to integrate OpenTelemetry Logs into an existing (non-OpenTelemetry) log stream.
+
+It should also be possible to bypass the OpenTelemetry Log API entirely and emit OpenTelemetry Logs (including Events)
+directly via existing language-specific logging libraries.
+
+OpenTelemetry will recommend that
+[instrumentation libraries](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/glossary.md#instrumentation-library)
+use the OpenTelemetry Log API to emit OpenTelemetry Events rather than using a generic logging library. This recommendation aims to provide users with a simple and consistent
+onboarding experience that avoids mixing approaches.
+
+OpenTelemetry will also recommend that application developers use the OpenTelemetry Log API to emit OpenTelemetry Events instead of using a generic
+logging library, as this helps prevent accidentally emitting logs that lack an event name or are unstructured.
+
+Recommending the OpenTelemetry Log API for emitting OpenTelemetry Events, rather than using a generic logging library, contributes to a clearer overall
+OpenTelemetry API story. This ensures a unified approach with first-class user-facing APIs for traces, metrics, and events,
+all suitable for direct use in native instrumentation.
+
+### Relationship to Span Events
+
+Events are intended to replace Span Events in the long term.
+Span Events will be deprecated to signal that users should prefer Events.
+
+Interoperability between Events and Span Events will be defined in the short term.
+
+### SDK
+
+This refers to the existing OpenTelemetry Log SDK.
+
+## Trade-offs and mitigations
+
+TODO
```

Comment:
I think that the trade-off is overloading of the "events" term which is already used in OpenTelemetry (Span Events as mentioned before).

The issue is that people may start confuse span events with log events.

What is the mitigation for avoiding the confusion among the users?

Reply:
isn't that addressed by this section?

> ### Relationship to Span Events
>
> Events are intended to replace Span Events in the long term. Span Events will be deprecated to signal that users should prefer Events.
>
> Interoperability between Events and Span Events will be defined in the short term.

Reply:
So maybe it will be better to move this section here.

Reply:
removed the empty sections

Reply:
> I think that on of the main trade-offs is overloading of the "events" term which is already used in OpenTelemetry for Span Events (as mentioned before).
> The issue is that people may start confuse span events with log events.

The long term point and the Sem Conv tooling is that there are and will be the same, so it becomes a transport (HOW) the event will be transmitted (eventually), because an event is an event.